app/models/ability.rb

```diff
@@ -0,0 +1,59 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    unless user.nil?
+      initialize_admin user and return if user.is_admin?
```

Comment:
Danke für den Hinweis, ich habs noch eingebaut!
Allerdings direkt in die initialize funktionen. Das geht sonst meines Erachtens nicht mehr wirklich aus dem Code hervor. Jetzt gibt es eine Loop, die die entsprechenden is-Methoden testet und bei positivem Rückgabewert die zugehörige init-Funktion ausführt. In den init-funktionen kann man so direkt definieren, dass ein admin zB ein wimi mit "diesen" besonderes rechten ist. danke für den tipp! :+1: